safehttp/form.go

```diff
@@ -93,15 +93,12 @@ func (f *Form) Bool(param string, defaultValue bool) bool {
 	if !ok {
 		return defaultValue
 	}
-	switch vals[0] {
-	case "true":
-		return true
-	case "false":
-		return false
-	default:
-		f.err = fmt.Errorf("values of form parameter %q not a boolean", param)
+	b, err := strconv.ParseBool(vals[0])
```

Comment:
The other functions use `paramVal` here. Idk which is more readable, but I think it makes the code more uniform to have the same name in all the functions.

Reply:
let's always use the same way to access params.